tests/queries/0_stateless/02841_array_remove.sql

```diff
@@ -0,0 +1,2 @@
+SELECT length(arrayRemove(x->(x%2==0), [1,2,3,4,5,6,7,8,9,10])) AS x;
```

Comment:
Missing tests for the normal mode of operation:

```
arrayRemove(array, element)
```

and the compatibility alias for PostgreSQL

Reply:
* pls add test Alexey asked for
* to learn how to add an alias check this [pr](https://github.com/ClickHouse/ClickHouse/commit/290c145acfc5accb2344b63356904af5f0a784ba)

Reply:
@rajatmohan22, need to add the required overload.